tests/dags/providers/provider_api_scripts/test_smithsonian.py

```diff
@@ -73,11 +73,11 @@ def test_validate_unit_codes_from_api_raises_exception(
     with patch.object(
         si,
         "get_new_and_outdated_unit_codes",
-        return_value=(set(new_unit_codes), set(outdated_unit_codes)),
+        return_value=(new_unit_codes, outdated_unit_codes),
     ):
         with pytest.raises(
             AirflowException,
-            match="^\n\*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary\**",
+            match="^\n\\*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary\\**",
```

Comment:
I think you could maybe use a [raw string](https://www.askpython.com/python/string/python-raw-strings) here so you don't have to escape the escape:
```suggestion
            match=r"^\n\*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary\**",
```